index.js

```diff
@@ -279,39 +326,61 @@ class instance extends instance_skel {
 		this.connect();
 	}
 
+	/**
+	 * Determine the internal identifier of the channel, according to the given channel, bus number and bus type
+	 * @param {int} channel
+	 * @param {char} bustype 
+	 * @param {int} bus 
+	 */
+	getFullChannelId(channel, bustype, bus) {
+		if (!bus) { bus = 1 };
```

Comment:
Hi, wouldn't be the following function definition not better?
`getFullChannelId(channel, bustype, bus = 1) {`

Reply:
Ah yes sure... 😅